Directory.Build.props

```diff
@@ -1,10 +1,11 @@
-<Project>
+﻿<Project>
   <Import
     Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), AspNetCoreSettings.props))\AspNetCoreSettings.props"
     Condition=" '$(CI)' != 'true' AND '$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), AspNetCoreSettings.props))' != '' " />
 
   <Import Project="version.props" />
   <Import Project="build\dependencies.props" />
+  <Import Project="build\sources.props" />
```

Comment:
Just a note since you were planning on bulk updating repos - I known Scaffolding tests attempt to copy the NuGet.config to restore test apps. You'd need to update it.

Reply:
Yup. We made changes in release/2.0.0 to handle this. I'll port those to dev now.